src/plone/restapi/serializer/dxcontent.py

```diff
@@ -80,7 +80,8 @@ def __call__(self, version=None, include_items=True):
         # Insert field values
         primary_field_name = self.get_primary_field_name()
         for schema in iterSchemata(self.context):
-            read_permissions = mergedTaggedValueDict(schema, READ_PERMISSIONS_KEY)
+            read_permissions = mergedTaggedValueDict(
```

Comment:
@tiberiuichim is it possible that you have flake8 with an 80 chars limit enabled in your IDE? We currently let black handle the line breaks and increased flake8 max-line-length to 200.